### ISSUE TITLE
Missing button label and decorative image present

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2009]
## Bug Report

**Problematic behavior**
The accessible name of the dropdown list opening button is the icon font name. This arrow_drop_down icon is decorative and should not be rendered.

**Possible Solution**
Before
```
<button class="c__button c__button--neutral c__button--neutral--tertiary c__button--nano c__select__inner__actions__open c__button--icon-only" type="button"><span class="c__button__icon"><span class="material-icons">arrow_drop_down</span></span></button>
```

After
``` 
<button class="c__button ... c__button--icon-only" type="button"
aria-label="Ouvrir la liste des formats"(…) <span class="c__button__icon">
<span class="material-icons" aria-hidden="true">arrow_drop_down</span>
</span></button>
```

**Critères RGAA**
Critère 7.1 :[ Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 
The user hears "arrow_drop_down", the text corresponds to the icon name and does not describe the action they are going to perform.

**Comment by Ovgodd** [comment]
PR : https://github.com/suitenumerique/cunningham/pull/379/
